exercise4B.sh

```diff
@@ -0,0 +1,15 @@
+#To return the gender, yearsExperience, and wage of the lowest earner to stdout
+
+#To return the genfer, yearsExperience, and wage of the lowest earner to stdout
+
+#To return number of female wage earners to stdout
+
+var1=$(cat $filename | sort -k4 -n | cut -d ',' -f1,2,4 | head -n 2 | tail -n 1)
```

Comment:
Sort doesn't work with csv format. You may want to replace "," with " " first: 
cat wages.csv | grep -v "gender" | tr "," " " | sort -n -k 4 | tail -n 1 | cut -d " " -f 1,2,4 | tr " " ","

-0.25 point